src/lib/list/list.scss

```diff
@@ -116,6 +118,26 @@ $mat-dense-list-icon-size: 20px;
     border-radius: 50%;
     padding: 4px;
   }
+
+  .mat-divider {
```

Comment:
can you separate the attributes from the rules here?

```scss
.mat-divider {
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%;

  [dir='rtl'] & {
    left: auto;
    right: 0;
  }

  &.mat-divider-inset {
    left: $mat-list-item-inset-divider-offset;
    width: calc(100% - #{$mat-list-item-inset-divider-offset});
    margin: 0;

    [dir='rtl'] & {
      left: auto;
      right: 72px;
    }
  }
}
```

Reply:
Fixed